packages/appcd-telemetry/src/telemetry.js

```diff
@@ -353,7 +354,10 @@ export default class Telemetry extends Dispatcher {
 				this.lastSend = Date.now();
 				scheduleSendEvents();
 			})
-			.catch(() => {});
+			.catch(e => {
+				this.lastSend = Date.now();
+				scheduleSendEvents();
```

Comment:
Overall, I like this mechanism.

At some point, I think we should create a more general daemon-wide event that is emitted when the network status changes just like a web browser (https://developer.mozilla.org/en-US/docs/Web/API/NavigatorOnLine/onLine#Example) or Titanium (https://docs.appcelerator.com/platform/latest/#!/api/Titanium.Network). I'll create a ticket.